packages/wallets/src/features/cfd/screens/Success/Success.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { ReactNode } from 'react';
+import { useActiveWalletAccount, useSortedMT5Accounts } from '@deriv/api';
+import { WalletGradientBackground } from '../../../../components/WalletGradientBackground';
+import { WalletMarketCurrencyIcon } from '../../../../components/WalletMarketCurrencyIcon';
+import { WalletText } from '../../../../components';
+import './Success.scss';
+
+type TSuccessProps = {
+    description: string;
+    marketType: Exclude<NonNullable<ReturnType<typeof useSortedMT5Accounts>['data']>[number]['market_type'], undefined>;
+    renderButton: () => ReactNode;
+    title: string;
```

Comment:
Made success modal to render the buttons, because there can be alot of button variants:
<img width="1063" alt="Screenshot 2023-10-17 at 2 39 30 PM" src="https://github.com/binary-com/deriv-app/assets/103016120/73764da1-7f2b-479b-b8f8-0311306ea1b1">